### ISSUE TITLE
Version 1.2.0 seems not to allow for opening or searching in Sourcegraph

### ISSUE THREAD

**Issue by emchap** [sourcegraph/sourcegraph-vscode-DEPRECATED#139]
I downloaded version 1.2.0 last night (running VSCode version 1.52.1, OSX 10.15.7, default browser set to Firefox), and the Open in Sourcegraph and Search in Sourcegraph commands don't seem to work. (Last night they were throwing errors, today nothing happens.)

Downgrading to v.1.1.0 works without incident.

Video walking through the behavior I'm seeing: https://share.getcloudapp.com/KouZx7B8

Apologies if I'm missing something silly but I wanted to highlight in case this isn't just me.

**Comment by matthewnitschke-wk** [comment]
Its not just you, encountering the same issue as well

For what its worth, when cloning the repo and building the extension locally, the issue does not appear to be present. Might be something wrong with the creation of the `.vsix` extension file

**Comment by pinkplus** [comment]
I have this stacktrace:

```
[2021-01-28 12:22:57.373] [exthost] [info] ExtensionService#_doActivateExtension sourcegraph.sourcegraph {"startup":false,"extensionId":{"value":"sourcegraph.sourcegraph","_lower":"sourcegraph.sourcegraph"},"activationEvent":"onCommand:extension.open"}
[2021-01-28 12:22:57.373] [exthost] [info] ExtensionService#loadCommonJSModule file:///home/yuxiao/.vscode-server/extensions/sourcegraph.sourcegraph-1.2.0/out/extension
[2021-01-28 12:22:57.385] [exthost] [error] Activating extension sourcegraph.sourcegraph failed due to an error:
[2021-01-28 12:22:57.386] [exthost] [error] Error: Cannot find module 'pump'
Require stack:
- /home/yuxiao/.vscode-server/extensions/sourcegraph.sourcegraph-1.2.0/node_modules/get-stream/index.js
- /home/yuxiao/.vscode-server/extensions/sourcegraph.sourcegraph-1.2.0/node_modules/execa/lib/stream.js
- /home/yuxiao/.vscode-server/extensions/sourcegraph.sourcegraph-1.2.0/node_modules/execa/index.js
- /home/yuxiao/.vscode-server/extensions/sourcegraph.sourcegraph-1.2.0/out/git.js
- /home/yuxiao/.vscode-server/extensions/sourcegraph.sourcegraph-1.2.0/out/extension.js
- /home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/loader.js
- /home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/bootstrap-amd.js
- /home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/bootstrap-fork.js
	at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
	at Function.Module._load (internal/modules/cjs/loader.js:686:27)
	at Function.t._load (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1116:657)
	at Function.i._load (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1085:334)
	at Function.n._load (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1081:567)
	at Module.require (internal/modules/cjs/loader.js:848:19)
	at require (internal/modules/cjs/helpers.js:74:18)
	at Object.<anonymous> (/home/yuxiao/.vscode-server/extensions/sourcegraph.sourcegraph-1.2.0/node_modules/get-stream/index.js:2:14)
	at Module._compile (internal/modules/cjs/loader.js:955:30)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
	at Module.load (internal/modules/cjs/loader.js:811:32)
	at Function.Module._load (internal/modules/cjs/loader.js:723:14)
	at Function.t._load (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1116:657)
	at Function.i._load (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1085:334)
	at Function.n._load (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1081:567)
	at Module.require (internal/modules/cjs/loader.js:848:19)
	at require (internal/modules/cjs/helpers.js:74:18)
	at Object.<anonymous> (/home/yuxiao/.vscode-server/extensions/sourcegraph.sourcegraph-1.2.0/node_modules/execa/lib/stream.js:3:19)
	at Module._compile (internal/modules/cjs/loader.js:955:30)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
	at Module.load (internal/modules/cjs/loader.js:811:32)
	at Function.Module._load (internal/modules/cjs/loader.js:723:14)
	at Function.t._load (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1116:657)
	at Function.i._load (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1085:334)
	at Function.n._load (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1081:567)
	at Module.require (internal/modules/cjs/loader.js:848:19)
	at require (internal/modules/cjs/helpers.js:74:18)
	at Object.<anonymous> (/home/yuxiao/.vscode-server/extensions/sourcegraph.sourcegraph-1.2.0/node_modules/execa/index.js:11:75)
	at Module._compile (internal/modules/cjs/loader.js:955:30)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
	at Module.load (internal/modules/cjs/loader.js:811:32)
	at Function.Module._load (internal/modules/cjs/loader.js:723:14)
	at Function.t._load (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1116:657)
	at Function.i._load (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1085:334)
	at Function.n._load (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1081:567)
	at Module.require (internal/modules/cjs/loader.js:848:19)
	at require (internal/modules/cjs/helpers.js:74:18)
	at Object.<anonymous> (/home/yuxiao/.vscode-server/extensions/sourcegraph.sourcegraph-1.2.0/out/git.js:35:33)
	at Module._compile (internal/modules/cjs/loader.js:955:30)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
	at Module.load (internal/modules/cjs/loader.js:811:32)
	at Function.Module._load (internal/modules/cjs/loader.js:723:14)
	at Function.t._load (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1116:657)
	at Function.i._load (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1085:334)
	at Function.n._load (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1081:567)
	at Module.require (internal/modules/cjs/loader.js:848:19)
	at require (internal/modules/cjs/helpers.js:74:18)
	at Object.<anonymous> (/home/yuxiao/.vscode-server/extensions/sourcegraph.sourcegraph-1.2.0/out/extension.js:38:15)
	at Module._compile (internal/modules/cjs/loader.js:955:30)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
	at Module.load (internal/modules/cjs/loader.js:811:32)
	at Function.Module._load (internal/modules/cjs/loader.js:723:14)
	at Function.t._load (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1116:657)
	at Function.i._load (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1085:334)
	at Function.n._load (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1081:567)
	at Module.require (internal/modules/cjs/loader.js:848:19)
	at require (internal/modules/cjs/helpers.js:74:18)
	at Function.r [as __$__nodeRequire] (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/loader.js:20:263)
	at p._loadCommonJSModule (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:1086:631)
	at p._doActivateExtension (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:982:742)
	at p._activateExtension (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:981:822)
	at Object.actualActivateExtension (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:979:453)
	at h._activateExtension (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:703:603)
	at /home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:703:281
	at Array.map (<anonymous>)
	at h._activateExtensions (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:703:269)
	at h.activateByEvent (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:700:745)
	at p._activateByEvent (/home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:980:300)
	at /home/yuxiao/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:989:111
	at processTicksAndRejections (internal/process/task_queues.js:94:5)
```

**Comment by pinkplus** [comment]
Pinging recent committers @tjkandala @felixfbecker @nicksnyder:

Could you have a look at this issue? After I upgrade to 1.2.0, I'm not even able to load the extension.

**Comment by nicksnyder** [comment]
@sourcegraph/web

**Comment by pinkplus** [comment]
I just want to mention that this issue is fixed in 1.2.2. I have been using 1.2.2 for several weeks and it's working pretty well. Thanks for the work! We can probably close this issue now.

**Comment by jeanduplessis** [comment]
Thank you for the confirmation @pinkplus
